spec/core/ics-004-channel-and-packet-semantics/UPGRADES.md

```diff
@@ -49,16 +50,27 @@ interface ChannelEnd {
   counterpartyChannelIdentifier: Identifier
   connectionHops: [Identifier]
   version: string
+  upgradeSequence: uint64
```

Comment:
This should be updated outside of the upgrades spec as well?

Reply:
Will do once this doc stabilizes